Makefile

```
@@ -3,7 +3,7 @@ all: lint test demo
 GO_BUILD_TAGS:=no_net,no_json,no_pprof
 
 demo:
-	go run  -tags $(GO_BUILD_TAGS) ./example/ -loglevel debug
+	go run  -tags $(GO_BUILD_TAGS) ./example/ -loglevel debug -only-valid
```

```suggestion
	go run -tags $(GO_BUILD_TAGS) ./example/ -loglevel debug -only-valid
```

I'm a crazy person 🤪

BTW, do you know unmake? 

Makefile linter
https://github.com/mcandre/unmake

---

I can tell :) fixing it, installed unmake but will ignore
```
warning: Makefile: MAKEFILE_PRECEDENCE: lowercase Makefile to makefile for launch speed
warning: Makefile:1: STRICT_POSIX: lead makefiles with the ".POSIX:" compliance marker, or else rename include files like *.include.mk
warning: Makefile:8: COMMAND_COMMENT: comment embedded inside commands will forward to the shell interpreter
warning: Makefile:14: COMMAND_COMMENT: comment embedded inside commands will forward to the shell interpreter
```
and
```
error: Makefile:3:15 found "=", expected: LF, comment, inline command, macro expansion, target, wait prerequisite marker
```
(it doesn't know about := ? uh?)

---

Dunno about unmake, but it helped me to spot some issues